django/forms/widgets.py

```diff
@@ -115,39 +117,54 @@ def __getitem__(self, name):
         raise KeyError('Unknown media type "%s"' % name)
 
     @staticmethod
-    def merge(list_1, list_2):
+    def merge(*lists):
```

Comment:
this is assuming you always get a list.
I would call them `iterables` and treat them as such. Therefore you will be only able to iterate over them once.

Reply:
Hmm. Is it commonplace to add arbitrary iterables in media definitions? Handling iterables isn't that much work but I think that in this case it's fine when the expectation is kept that media consists of JS and CSS tuples/lists. Or not?